src/components/CardList.js

```diff
@@ -0,0 +1,14 @@
+import CardItem from './CardItem';
+import './CardList.css';
+
+const CardList = ({ links }) => {
+  return (
+    <div className="CardList">
+      {links.map(list => (
+        <CardItem key={list.id} {...list} />
+      ))}
+    </div>
```

Comment:
CardList 컴포넌트는 카드를 여러개 리스트 형식으로 보여주고 있는데요~
ul 태그를 사용해서 겉에를 감싸주고, CardItem 컴포넌트는 li 태그로 사용해서
ul-li 구조가 되면 좋을 것 같아요~
```suggestion
    <ul className="CardList">
      {links.map(list => (
        <CardItem key={list.id} {...list} />
      ))}
    </ul>
```